scribble-doc/scriblib/scribblings/autobib.scrbl

```diff
@@ -90,6 +92,29 @@ section for the bibliography. It has the contract
 (->* () (#:tag string? #:sec-title string?) part?)
 ]
 
+If provided, the function bound to @racket[cite-author]
```

Comment:
I'm fine with that. Would it be okay to have it in both places actually?

Also fine to your second point.